web/source/osk/visualKeyboard.ts

```diff
@@ -1304,7 +1326,12 @@ namespace com.keyman.osk {
     private getVerticalLayerGroupPadding(): number {
       // For touch-based OSK layouts, kmwosk.css may include top & bottom padding on the layer-group element.
       const computedGroupStyle = getComputedStyle(this.layerGroup.element);
-      return parseInt(computedGroupStyle.paddingTop, 10) + parseInt(computedGroupStyle.paddingBottom, 10);
+      let pt = parseInt(computedGroupStyle.paddingTop, 10);
+      let pb = parseInt(computedGroupStyle.paddingBottom, 10);
+
+      pt = isNaN(pt) ? 0 : pt; // parseInt('') == NaN; handle such cases!
+      pb = isNaN(pb) ? 0 : pb;
```

Comment:
I think we can [shorten this](https://stackoverflow.com/a/6736521/1836776) with `||`:

```suggestion
      const pt = parseInt(computedGroupStyle.paddingTop, 10) || 0; // parseInt('') -> NaN, fallback to 0
      const pb = parseInt(computedGroupStyle.paddingBottom, 10) || 0;
```